webapp/src/main/resources/application.sql.yml

```diff
@@ -40,6 +51,14 @@ sql:
       from school
       where (1=:statewide or district_id in (:district_ids) or id in (:school_ids))
 
+    exists: >-
+      select exists(
+        select 1
+        from school
+        where id = :id
+          and (1=:statewide or district_id in (:district_ids) or id in (:school_ids))
```

Comment:
Good point. You are just searching to match 1 indexed ID so it will make no difference 👍